test_version/version_specific_tests_dir/twitter_scrooge/test_external_repo_versions.bzl

```diff
@@ -0,0 +1,20 @@
+def external_dependency_version_tests():
```

Comment:
did you try doing this in the BUILD file? I think you can do a for loop there and then we can drop this indirection but I might be remembering it wrong

Reply:
I just tried, I get this error:
```
ERROR: .../rules_scala/test_version/test_scrooge_version_1591969888/twitter_scrooge/BUILD:12:1: for loops are not allowed at the top level. You may move it inside a function or use a comprehension, [f(x) for x in sequence]
```

I'd personally prefer to keep the indirection instead of hand-rolling the targets or trying to fit this code in a for comprehension, but I'd love to hear what you think.

Reply:
I'd prefer trying a for comprehension but will definitely live with what you sent

Reply:
It definitely looks good with a for comprehension. Thanks for insisting :)